package.json

```diff
@@ -59,7 +59,8 @@
     "test:unit": "haste-playable test --mocha",
     "pretest:playback": "haste-playable clean && npm run bundle:playback-test && npm run bundle:playback-test-with-adapters",
     "test:playback": "haste-playable test --karma dev-env/karma.conf.js && haste-playable test --karma dev-env/karma-with-adapters.conf.js",
-    "precommit": "lint-staged && npm run test:unit && npm run test:node && npm run documentation:gitadd",
+    "precommit": "lint-staged",
+    "prepush": "npm run test:unit && npm run test:node && npm run documentation:gitadd",
```

Comment:
`npm run documentation:gitadd` should be in `precommit` (code diff => doc diff)

Reply:
Tests on hooks mb remove at all and allow PR to run them (+disallow to push in master)

Reply:
A side effect of "disallow push in master" that we could not push release commits/tags from master